docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,126 @@
+Proctored Exams
+===============
+
+Helpful ReadTheDocs Pages:
+
+- `Configuring a Proctoring Provider <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/proctored_enabling.html#configuring-proctoring-provider>`_
+- `Creating a Proctored Exam <https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/proctored_exams/pt_create.html#creating-a-proctored-exam>`_
+
+Prerequisite steps:
+    #. Use Google Chrome
+    #. Have a staff account
+    #. Have a non-staff learner account enrolled in the verified track for this course
+    #. As your non-staff user in your current window, and enroll in the course where you have set up your proctored exam
+    #. In an incognito window, login to your staff account and visit your Support Tools page and search in the email for your non-staff user. You should now see the option to change the enrollment track of that user. Change the enrollment track of the user to ""verified"" (choose any reason when prompted, it doesn't matter)
+    #. In that same incognito, still signed in as staff, navigate to the Instructor dashboard
+    #. Find or create a test course within Stage with proctoring enabled and an LTI proctoring provider chosen, certificates are enabled, and where the exams IDA waffle flag is enabled
+        #. The exams IDA waffle flag can be found/created in the LMS' Django Admin under "Waffle Flag Course Overrides"
+        #. If needed, you can make a new entry using "course_apps.exams_ida" as the waffle flag, the course ID, and by setting it to "Enabled"
+    #. Find or create two exam subsections with a type of 'Proctored exam' in your test course
+        #. This can be done by creating a subsection and clicking the 'Configure' button (should look like a gear), going to the "Advanced Tab", and selecting "Proctored"
+        #. You MUST also set this subsection to be Graded as a Midterm or Final exam
+        #. This subsection should contain at least one unit with one gradeable block, e.g. a generic multiple choice question in order to test downstream effects to the gradebook
+    #. You should also have a tab open with your event bus logs
+
+
+Proctored Exam Flow
```

Comment:
I think these another set of cases here for submitting an attempt that is auto verified and one needs a second review.

Reply:
I still think we're missing these test cases ensuring a review by the proctoring provider is received and the attempt is updated. We kind of do that as a side effect of setup for testing the instructor dashboard but I think it's best to be explicit. If someone is just testing exam flow they should be validating this case.

Reply:
I added a test to the Proctorio section about checking that a review of the exam exists. Regardless of whether or not we're using Proctorio, wouldn't the test that we received a review by a proctoring provider be checking the exams dashboard itself?

Reply:
I think the test I'm looking for is something that validates the ACS works as intended. So a test along the lines of "if a user submits an attempt without violation the proctoring software can update the attempt to verified" / "if a user submits with violations the proctoring software can update the attempt to review required".

I think that's a distinctly different test from testing that a verified/rejected/second review attempt can be reset in the UI since that testing instructor experience but the ACS is part of the learner / exam flow.